nototools/notoconfig.py

```diff
@@ -40,20 +40,21 @@ def _setup():
   values will hold a mapping from the <name> to value.
   Blank lines and lines starting with '#' are ignored."""
 
-  configfile = path.expanduser("~/.notoconfig")
-  if path.exists(configfile):
-    with open(configfile, "r") as f:
-      for line in f:
-        line = line.strip()
-        if not line or line.startswith('#'):
-          continue
-        k, v = line.split('=', 1)
-        values[k.strip()] = v.strip()
-  else:
-    # This needs to be silent.  It causes a makefile error in noto-emoji,
-    # which expects stdout to consist only of the output of a python
-    # script it runs.
-    pass
+  roots = ['~', '/usr/local/share/noto']
+  for root in roots:
+    configfile = path.expanduser(path.join(root, '.notoconfig'))
```

Comment:
A dot file in /usr/local/share/noto is a bit odd, isn't it?  How about /usr/local/share/noto/notoconfig or /usr/local/share/noto/config ?